packages/client/wallets/aa/package.json

```diff
@@ -2,7 +2,7 @@
     "author": "Paella Labs Inc",
     "dependencies": {
         "@ambire/signature-validator": "1.3.1",
-        "@crossmint/common-sdk-base": "0.0.4",
+        "@crossmint/common-sdk-base": "0.0.5",
```

Comment:
This does not exist yet. This PR needs to be merged and released!
https://github.com/Crossmint/crossmint-sdk/pull/398